LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MultiGatewayFrameCounterUpdateStrategy.cs

```diff
@@ -24,12 +24,7 @@ public async Task<bool> ResetAsync(LoRaDevice loRaDevice, uint fcntUp, string ga
 
             loRaDevice.ResetFcnt();
 
-            if (await InternalSaveChangesAsync(loRaDevice, force: true))
-            {
-                return await this.loRaDeviceAPIService.ABPFcntCacheResetAsync(loRaDevice.DevEUI, fcntUp, gatewayId);
-            }
-
-            return false;
+            return await this.loRaDeviceAPIService.ABPFcntCacheResetAsync(loRaDevice.DevEUI, fcntUp, gatewayId);
```

Comment:
now it could happen we reset the function but not the twins. what could be the side effect of that? I don't think I see a problem with this.